src/udp_load_balancer.h

```diff
@@ -1,2 +1,25 @@
+#ifndef _DPLB_UDP_LOAD_BALANCER_
```

Comment:
can you explain why is this needed?

Reply:
It is standard practice not to include header file twice.
for example src/udp_load_balancer.c has:

#include "udp_load_balancer.h"
#include "argparse.h"

but src/argparse.c also has
#include "udp_load_balancer.h"